Makefile

```diff
@@ -326,6 +326,7 @@ test-local:
 	CGOENABLED=0 go test -count=1 -cover -short $$(go list ./src/server/... | grep -v '/src/server/pfs/fuse') -timeout $(TIMEOUT)
 
 test-auth:
+	./etc/testing/start_postgres.sh
```

Comment:
Auth tests now require postgres as a backing store for the embedded Dex server.

Reply:
Are the auth tests separate from the identity tests? Both require postgres, or just auth?

Reply:
Auth tests rely on the identity server, which relies on postgres.